### PR TITLE
Projects list search

### PR DESCRIPTION
**Pull request by scrummish**
Hey yall, this is the search bar component.
Let me know your thoughts!

### CODE REVIEW

File: pages/projects/index.js [hunk 1]
```diff
@@ -1,77 +1,87 @@
...
-          projects.map((project) => {
-            return (
-              <ProjectCard project={project} key={`project-${project.slug}`}/>
-            )
-          })
-        }
+      <SearchBar
+        searchQuery={searchQuery}
+        setSearchQuery={setSearchQuery}
+        options={projectsFiltered}
```
Comment (Woozl) [inline pages/projects/index.js:53]: I think the autocomplete has the search and filtering functionality built-in to display the list of matching items, so I don't think it makes sense to filter the options? I may be wrong though.

**Review by Woozl — COMMENTED** [review]
Looks pretty good. There seems to be a slight bug if I reload the page and then search an item and press enter on it in the list, which then displays the whole unfiltered list:

https://github.com/mbwatson/renci-dot-org/assets/16181779/8bde9fd8-2749-45e7-9e86-0ef2451ab92b

I wonder though if the autocomplete is needed here. I think a simple text input that filters and sorts the list would probably suffice, especially if the matching text could be highlighted or bolded in the project cards.

**Comment by mbwatson** [comment]
good bug-find, @Woozl. i missed that.

i'm wondering, too, now about autocomplete here. it seems that the filtered cards and filtered terms are the same list, so a simple filter/search would likely suffice.

**Review by mbwatson — CHANGES_REQUESTED** [review]
after some thought, @scrummish, let's drop the autocomplete (as it's redundant functionality in this instance), and let's just have a plain ol' text input filter situation. can you make this adjustment?

**Review by mbwatson — CHANGES_REQUESTED** [review]
looks good. i think it's cleaner, but i think there's more to clean out with autocomplete gone. i don't see a difference regardless of whether `StyledAutocompleteSearch` is used. additionally, that `options` prop is now obsolete.

lastly, there's a `fullWidth` is prop available on `FormControl`, so the `width: '100%'` stuff can go away if that's used.

these changes should simplify the component and clean out the leftover autocomplete artifacts.

**Review by Woozl — APPROVED** [review]
👍

**Pull request merged**